.github/workflows/deploy.yaml

```diff
@@ -50,6 +50,7 @@ jobs:
 
       - uses: pypa/gh-action-pypi-publish@v1.5.0
         name: Deploy to Test PyPi
+        if: github.event_name == 'push'
         with:
           user: __token__
           password: ${{ secrets.test_pypi_password }}
```

Comment:
Also, adding this line could help to avoid errors:
```
          skip_existing: true
```

Reply:
that's true, I'll put that in.